proxy/envoy/ingress.go

```diff
@@ -161,17 +159,19 @@ func generateIngress(conf *IngressConfig) *Config {
 		},
 	}
 
-	// configure for HTTPS if provided with a secret name
-	if conf.Secret != "" {
-		// configure Envoy
+	// configure for HTTPS if there is a secret
+	if secret, err := conf.Secrets.GetSecret("*"); err != nil {
```

Comment:
I was intending to extend it that way in a follow-on PR, but I can add it to this PR instead.